makeflow/src/makeflow.c

```diff
@@ -1669,6 +1680,43 @@ int main(int argc, char *argv[])
 		makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
 	}
 
+	if (batch_queue_type == BATCH_QUEUE_TYPE_MESOS) {
+		pid_t mesos_PID;
+		mesos_PID = fork();			
+		char *mesos_cwd;
+		mesos_cwd = path_getcwd();
+
+		char *mesos_py = "/afs/nd.edu/user37/ccl/software/external/mesos-0.26.0/amd64_linux26/lib/python2.6/site-packages";
+		char *cctools_path = getenv("CCTOOLS");
+		char *exe_py_path = string_format("%s/bin/mf_mesos_scheduler", cctools_path);
+		char *envs[] = {"LD_PRELOAD=/afs/nd.edu/user37/ccl/software/external/gcc-4.9.3/amd64_linux26/lib64/libstdc++.so.6:/afs/nd.edu/user37/ccl/software/external/svn-1.9.4/amd64_linux26/lib/libsvn_delta-1.so", "CCTOOLS=/afs/crc.nd.edu/user/c/czheng2/cctools", NULL};
```

Comment:
Ok, this is what you needed to do in order to run mesos in our local environment.
But, I don't think it's the general case, esp when people install in /usr/bin.
I think you should consider this to be a setting peculiar to Notre Dame and just set it manually before you run Makeflow.